extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -369,6 +369,11 @@ public function getData() {
 				$community = $this->communityDataService->getCommunityData();
 				if ( !empty( $community ) ) {
 					$community[ 'community_data' ] = 'true';
+
+					if ( !empty( $community[ 'image_id' ] ) ) {
+						$url = CuratedContentHelper::getImageUrl( $community[ 'image_id' ] );
```

Comment:
Maybe you could assign it to community['image_url'] directly? What about using one of those, CuratedContentController::extendWithImageData or CuratedContentHelper::findImageUrl.